source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/locale-input/-utils/helpers.tsx

```diff
@@ -16,13 +16,15 @@ export const getSuggestions = (data: ILanguageInfo[]) => (value) => {
   const has = hasSearchTerm(value);
 
   return data.filter(
-    ({ name, nameInLocale, region, tag, localname, regions, names }) =>
+    ({ name, nameInLocale, region, tag, localname, regions, names, variants }) =>
       has(nameInLocale) ||
       has((names || []).join()) ||
       has(localname || name) ||
-      has(regions || region) ||
+      has((regions || []).join()) ||
```

Comment:
this is not really an array at runtime see comment below

Reply:
I believe it is an array now.